README.md

```diff
@@ -42,14 +55,66 @@ var output = postcss([
 console.log(output);
 ```
 
-[*For more general PostCSS usage, see this section*](https://github.com/postcss/postcss#usage)
+# Syntax
+
+### Defining Custom Properties: the `--*` family of properties
```

Comment:
Mostly done. I did add some of the other sections back, but the original explanation was a bit incomplete IMHO.